src/main/resources/library/attribute-definitions.json

```diff
@@ -4,14 +4,16 @@
   "title": "Library attribute definitions, v1",
   "description": "Constraints, facet definitions, and display definitions for FireCloud Library",
   "type": "object",
-  "required": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian",
-               "library:datasetDepositor", "library:datasetOwner", "library:institute", "library:indication", "library:numSubjects", "library:projectName", "library:datatype",
+  "required": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian", "library:datasetDepositor",
+               "library:datasetOwner", "library:institute", "library:numSubjects", "library:projectName", "library:datatype",
```

Comment:
Need to talk to Moran about this, I think I might have gotten confused / shouldn't have made this change.